tests/models/llava_next/test_modeling_llava_next.py

```diff
@@ -12,7 +12,7 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-""" Testing suite for the PyTorch Llava-NeXT model. """
+"""Testing suite for the PyTorch Llava-NeXT model."""
```

Comment:
Would be great to add a slow integration test similar to the one we have for llava: https://github.com/huggingface/transformers/blob/fbb41cd4209c00671fef583a39db2f608fcd7098/tests/models/llava/test_modeling_llava.py#L488

Reply:
Can you help add it, I'm not familiar with llava's tests

Reply:
I can add it, no worries 😄